query-engine/connector-test-kit-rs/query-engine-tests/tests/new/regressions/prisma_7434.rs

```diff
@@ -1,6 +1,6 @@
 use query_engine_tests::*;
 
-#[test_suite(schema(autoinc_id), capabilities(CreateMany, AutoIncrement), exclude(CockroachDb))]
+#[test_suite(schema(autoinc_id), capabilities(AutoIncrement), exclude(CockroachDb))]
```

Comment:
Random find, a couple of line below D1 is excluded, just wanted to let you know, would it be interesting to change that?
>#[connector_test(exclude(CockroachDb, Sqlite("cfd1")))]